lib/capistrano/asg/tasks/rolling.rake

```diff
@@ -168,4 +173,31 @@ namespace :rolling do
       end
     end
   end
+
+  desc 'Get status of instance refresh'
+  task :instance_refresh_status do
+    groups = {}
+    config.autoscale_groups.each { |group| groups[group.name] = group }
+    while groups.count.positive?
+      groups.each do |name, group|
+        refresh = group.latest_instance_refresh
+        status = refresh[:status]
+        percentage_complete = refresh[:percentage_complete]
+        refresh_completed = Capistrano::ASG::Rolling::AutoscaleGroup::COMPLETED_REFRESH_STATUSES.include?(status)
```

Comment:
You can maybe include `completed` in the return of `group.latest_instance_refresh`, or add another method like `group.refresh_completed?`